app/estimate_square_size_test.go

```diff
@@ -207,3 +207,11 @@ func Test_maxIndexWrapperOverhead(t *testing.T) {
 		assert.LessOrEqual(t, len(wtx)-len(tx), wrapperOverhead+indexOverhead, i)
 	}
 }
+
+func txsToBytes(txs coretypes.Txs) [][]byte {
+	e := make([][]byte, len(txs))
+	for i, tx := range txs {
+		e[i] = []byte(tx)
+	}
+	return e
+}
```

Comment:
note: this was moved out from the shares package because no one was using it. IIRC there's a similar function in core we could just use